Sources/FoundationEssentials/_ThreadLocal.swift

```diff
@@ -24,7 +24,7 @@ internal import threads
 #endif
 
 struct _ThreadLocal {
-#if canImport(Darwin) || os(Android) || canImport(Glibc)
+#if canImport(Darwin) || canImport(Bionic) || canImport(Glibc)
```

Comment:
Here we are using `canImport(Bionic)` in place of `os(Android)` but we are using `canImport(Android)` above.